pandas/core/indexes/timedeltas.py

```diff
@@ -176,7 +176,6 @@ def _join_i8_wrapper(joinf, **kwargs):
 
     _freq = None
 
-    _box_func = TimedeltaArray._box_func
```

Comment:
why is this removed?

Reply:
Same issue again already defined as a class property